sphinxcontrib/jupyter/writers/translate_all.py

```diff
@@ -597,13 +602,21 @@ def depart_raw(self, node):
     # =============
 
     def visit_jupyter_node(self, node):
-        if node['cell-break']:
+        if 'cell-break' in node.attributes:
```

Comment:
@FelipeMaldonado does this need to track `True/False` values of the option or is it just tracking `:cell-break:` as an option?

Reply:
In this particular case, they're equivalent. Since when the option is included, it's defined with a True value.